ext/tidy/tidy.c

```diff
@@ -304,7 +304,12 @@ static void php_tidy_quick_repair(INTERNAL_FUNCTION_PARAMETERS, bool is_file)
 	}
 
 	if (ZEND_SIZE_T_UINT_OVFL(ZSTR_LEN(data))) {
-		zend_argument_value_error(1, "is too long");
+		if (is_file) {
+			zend_string_release_ex(data, false);
+			zend_argument_value_error(1, "Input string is too long");
```

Comment:
This is possibly better wording?
```suggestion
			zend_argument_value_error(1, "File content is too long");
```

Reply:
Then we should change it consistently for the other messages too I guess, but that may be more like master-branch material?

Reply:
Right, leave it as a follow-up PR then :)